templates/package.json.tpl

```diff
@@ -10,6 +10,9 @@
     "conventional-changelog-conventionalcommits": "^6.0.0",
 {{- if has "node" (stencil.Arg "grpcClients") }}
     "hjson": "^3.2.2",
+{{- end }}
+{{- range stencil.GetModuleHook "nodejs_dependencies" }}
```

Comment:
gotcha, I'm _slightly_ concerned because we really wanted to get rid of `nodejs` as being part of stencil repos eventually... (only semantic-release needs it today) and that the version is meant to be exclusively owned by stencil.

I don't think it's enough to hard block on it, but I would like to get my team's opinion on it first before going forward with this (just in case I'm missing anything!)

Reply:
All right, let me know in which way we should go in case we cannot use this file

Reply:
@voslartomas We're fine with it! We're going to just selectively render the `package.json` in the future, once we remove `semantic-release`. At that point we might talk about ownership of node for the whole repo, but that's quite a ways off :)